### ISSUE TITLE
add option to use a monospace font in edit mode (and/or view mode)

### ISSUE THREAD

**Issue by legrostdg** [billthefarmer/diary#156]
It would be nice to be able to have a setting to use a monospace font in edit mode, especially for writing code. Having another option for using a monospace font in view mode could also be useful.

**Comment by billthefarmer** [comment]
It's a diary, not a code editor. You are probably using the wrong app. If you want an editor with various font options, themes, and code highlighting, use my [Editor](https://github.com/billthefarmer/editor/) app.

**Comment by legrostdg** [comment]
I mentioned "writing code", but of course I did not mean replacing an editor. I was more thinking about small snippets that one can put in a diary. Code highlighting, etc. is bonus for code, but it can be really hard to follow even small snippets without a mono font.

I wanted to use utf-8 box drawing characters https://www.w3schools.com/charsets/ref_utf_box.asp but it is difficult to use without a mono font.

I guess there are a lot of other use case (pseudo-tables, etc.) that are easier with a mono font.

**Comment by billthefarmer** [comment]
You already get a monospaced font for code blocks, because the generated text is put inside `<pre>` and `<code>` blocks.

**Comment by legrostdg** [comment]
OK
